l10n_es_aeat_sii/wizards/send_first_semester.py

```diff
@@ -35,8 +35,8 @@ def execute(self):
             out_invoices.with_context(override_eta=0).send_sii()
         if self.in_invoices:
             in_invoices = self.env['account.invoice'].\
-                search([('date_invoice', '<=', self.date_to),
-                        ('date_invoice', '>=', SII_SEMESTER_DATE_INI),
+                search([('period_id.date_start', '<=', self.date_to),
+                        ('period_id.date_stop', '>=', SII_SEMESTER_DATE_INI),
```

Comment:
Aunque creo que no afecta, es más correcto [('period_id.date_stop', '<=', self.date_to), ('period_id.date_start', '>=', SII_SEMESTER_DATE_INI)]

Reply:
He preferido hacerlo así, mas amplio para evitar que si se pone una fecha de enero, no envíe ninguna factura recibida, pero en todo caso puedo ponerlo como prefiráis